pkg/csi_driver/node.go

```diff
@@ -100,10 +106,17 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	}
 	defer s.volumeLocks.Release(targetPath)
 
+	vs, ok := s.volumeStateStore[targetPath]
```

Comment:
maybe put a comment about why we chose targetPath as the key ? target path corresponds to the pods and pods may belong to different namespace and would need its own access check.

Reply:
Done.